### PR TITLE
Add Makefile with build and test targets

### LINKED ISSUE #19: Add makefile for build/e2e tests/unit tests

### PR DESCRIPTION
**Pull request by cutwater**
Closes #19

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+$(BUILD_DIR)/ipcontroller: _gopath = /gopath
+$(BUILD_DIR)/ipcontroller: \
+	_directory = $(_gopath)/src/github.com/Mirantis/k8s-externalipcontroller/
+$(BUILD_DIR)/ipcontroller: $(BUILD_DIR)
+	docker run --rm -v "$(ROOT_DIR):$(_directory)" -e "GOPATH=$(_gopath)" \
+		-w "$(_directory)" $(GOLANG_IMAGE) go build -o $@ cmd/ipcontroller.go
+
+
+$(BUILD_DIR)/e2e.test:
+	go test -c -o $(BUILD_DIR)/e2e.test ./test/e2e/
```
Comment (dshulyak) [inline Makefile]: for now we need to run both e2e and integration with sudo

Comment (dshulyak) [inline Makefile]: good idea

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+clean:
+	rm -rf $(BUILD_DIR)
+
+
+$(BUILD_DIR):
+	mkdir -p $(BUILD_DIR)
+
+
+$(BUILD_DIR)/ipcontroller: $(BUILD_DIR)
+	go build -o $@ cmd/ipcontroller.go
```
Comment (dshulyak) [inline Makefile]: so you don't want to use container to build binary?

Comment (cutwater) [inline Makefile]: It is not necessary to build it inside container. I'm thinking about adding containerized targets to build binaries inside docker container.

File: prepare.sh [hunk 3]
```diff
@@ -9,6 +9,8 @@ NUM_NODES=${NUM_NODES:-2}
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 WORKDIRECTORY=${WORKDIRECTORY:-}
 
+K8S_ENDPOINT="http://localhost:4001"
```
Comment (dshulyak) [inline prepare.sh]: it should be ETCD_ENDPOINT

**Pull request merged**